src/manifest-creation.ts

```diff
@@ -47,8 +47,7 @@ export class ManifestCreation {
       name: manifest.name || pkg.name,
       public: manifest.public || true,
       redirect_url: `${baseUrl}/probot/setup`,
-      // TODO: add setup url
-      // setup_url:`${baseUrl}/probot/success`,
+      setup_url:`${baseUrl}/probot/success`,
```

Comment:
How would you feel about redirecting to the repo on which they just installed the GitHub App? 

I feel like going to the repo is the next logical step (since folks want to start playing around with their app) and it's a surprising amount of effort to get to the repo at the moment.

Reply:
Unfortunately, since the manifest is passed _before_ they install the app, we can't know what that repo will be. :/

Reply:
🤦‍♂️

Reply:
Is there a way for us to achieve the intended "redirect to the repo" behaviour in the following way:

1. Set the `setup_url` to be `${baseUrl}/probot/success` (as it is already in this PR)
2. When the webhook comes in that tells us on which repo the app was installed on, somehow persist that value
3. When the user is redirected to `${baseUrl}/probot/success`, retrieve the persisted repo value, and redirect them to that repo

Would be an awesome win for user on-boarding if this was possible 😍